lib/sysfs_attr.c

```diff
@@ -307,6 +307,7 @@ static struct sysfs_attribute *add_attribute_to_list(struct dlist *alist,
 	}
 
 	if (!alist) {
+		sysfs_close_list(alist);
```

Comment:
`alist` is already `NULL` here, calling `sysfs_close_list` on an empty `alist` is a no-op.

Reply:
I removed this.